Mlw fix release 10.13 10.15 (#114)
SoftwareAG/c8y-releasenotes: content/release-10-15-0/announcements-10-15-0.md

Further to the CEL (Esper) deprecation notice in [release 10.5](/release-10-5-0/migration-10-5-0/) and the subsequent announcement of end of support in [release 10.7](/release-10-7-0/announcements-10-7-0/) it has been possible to continue using Esper in an unsupported mode. With effect from release 10.15 the CEL (Esper) functionality has been removed completely and cannot be used anymore.

If you need assistance to migrate your streaming analytics logic from Esper to Apama, please contact [Software AG Global Support](/about/contacting-support/).

> ### Machine learning Workbench

Reviewer: ```suggestion
### Machine Learning Workbench
```

- The following APIs are only accessible to  the "Machine Learning Admin" user group now:

   GET - jnb-sessions:
    
    ```
    {{url}}/service/mlw/jnb-sessions
    ```
   GET - jnb-content:
    
    ```
    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourceID}}/jnb-content
    ```
   PUT - jnb-content:
    
    ```
    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourceID}}/jnb-content
    ```
   
   POST - training workflows:
    
    ```
    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourcesID}}/workflow
    ```
   POST - code execution:
    
    ```
    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourcesID}}/execute
    ```
   POST - neural networks:
    
    ```
    {{url}}/service/mlw/projects/{{projectID}}/resources/{{resourcesID}}/trainNN
    ```

- The "Machine Learning Manager" user group has been restricted from executing Jupyter notebooks and accessing the assets section as well.

- The "Machine Learning User" user group has been restricted from accessing the assets section.

- This access limitation will be a breaking change for the existing "Machine Learning Manager" user group.

> - Those users who are currently part of "Machine Learning Manager" group and needs code execution access will need to be a part of higher privileged group, i.e. "Machine Learning Admin".

Reviewer: ```suggestion
- Those users who are currently in the "Machine Learning Manager" group and need code execution access must be accordingly added to a higher privileged group, for example, "Machine Learning Admin".
```
